specs/data_structures.md

```diff
@@ -723,11 +723,55 @@ Wrapper for evidence data.
 
 #### Evidence
 
+#### DuplicateVote
+
 | name     | type                    | description |
 | -------- | ----------------------- | ----------- |
 | `voteA`  | [Vote](#vote)           |             |
 | `voteB`  | [Vote](#vote)           |             |
 
+type LightClientAttackEvidence struct {
+ ConflictingBlock *LightBlock
+ CommonHeight     int64
+}
+
+#### LightClientAttack
+
+| name     | type                    | description |
+| -------- | ----------------------- | ----------- |
+| `conflictingBlock`  | [LightBlock](#lightblock)         |             |
+| `commonHeight`      | int64           |             |
+
+#### LightBlock
+
+| name     | type                    | description |
+| -------- | ----------------------- | ----------- |
+| `signedHeader`  | [SignedHeader](#signedheader)           |             |
+| `validatorSet`  | [ValidatorSet](#validatorset)           |             |
+
+#### SignedHeader
+
+| name     | type                    | description |
+| -------- | ----------------------- | ----------- |
+| `Header`  | [Header](#header)           |             |
+| `commit`  | [commit](#commit)           |             |
+
+#### ValidatorSet
+
+| name     | type                    | description |
+| -------- | ----------------------- | ----------- |
+| `validators`  | [Array of Validator](#validator)           |             |
+| `proposer`  | [Validator](#validator)           |             |
+
+<!-- #### Validator
+
+| name     | type                    | description |
+| -------- | ----------------------- | ----------- |
+| `address`  | [Address](#address)           |             |
+| `pubKey`  | [publickey](#publickey)           |             |
+| `VotingPower`  | int64           |             |
+| `ProposerPriority`  | int64           |             | -->
```

Comment:
@adlerjohn within this page you have a validator type, but the type you have defined differs from this one and the one that will be used within ll-core. How would you prefer to handle this?

This may tie into the conversation on how to define ll-core specific types.